src/ai/backend/manager/models/image.py

```diff
@@ -294,8 +295,19 @@ def _get_image_endpoint_join_condition():
     return ImageRow.id == foreign(EndpointRow.image)
 
 
+class ImageStatus(enum.StrEnum):
+    ALIVE = "ALIVE"
```

Comment:
I'm not sure `ALIVE` state is good naming..

Reply:
How about `Ready`?

Reply:
IMHO, the name "ready" makes it seem like the image should be immediately usable.
However, when we try to create a session with a "ready" image, we still need to pull the image from the registry first.

Reply:
Should I rename this to `Ready` 👀? @HyeockJinKim

Reply:
Determined as `ALIVE`.